org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/CompletionHandlerTest.java

```diff
@@ -1227,6 +1227,37 @@ public void testCompletion_testClassesDontLeakIntoMainCode() throws Exception {
 		assertEquals("Test proposals leaked:\n" + list.getItems(), 0, list.getItems().size());
 	}
 
+	@Test
+	public void testCompletion_testMethodWithParams() throws Exception {
+		ICompilationUnit unit = getWorkingCopy(
+		//@formatter:off
+		"src/org/sample/Test.java",
+		"package org.sample;\n" +
+		"public class Test {\n" +
+		"	public static void main(String[] args) {\n" +
+		"		fo\n" +
+		"		System.out.println(\"Hello World!\");\n" +
+		"	}\n\n" +
+		"	/**\n" +
+		"	* This method has Javadoc\n" +
+		"	*/\n" +
+		"	public static void foo(String bar) {\n" +
+		"	}\n" +
```

Comment:
please also add another foo method, with different parameters and javadoc, to ensure the proposals return different documentations

Reply:
I have updated the PR.